lib/matrix_api_lite/generated/model.dart

```diff
@@ -1703,6 +2291,26 @@ class LoginResponse {
   /// optionally validating the URLs within. This object takes the same
   /// form as the one returned from .well-known autodiscovery.
   DiscoveryInformation? wellKnown;
+
+  @override
+  bool operator ==(Object other) =>
+      other is LoginResponse &&
+      other.accessToken == accessToken &&
```

Comment:
We probably shouldn't log this via toString by default either.

Reply:
removed toString